js/SaveInterface.js

```diff
@@ -327,6 +327,13 @@ function SaveInterface(PlanetInterface) {
     }
 
     this.afterSaveLilypondLY = function(lydata, filename){
+        let tmp = jQuery("<textarea />").appendTo(document.body);
+        tmp.val(lydata);
+        tmp.select();
+        tmp[0].setSelectionRange(0, lydata.length);
+        document.execCommand("copy");
+        tmp.remove();
+        textMsg("Copied Lilypond code to clipboard. You can play it at <a href='http://lilybin.com' target='_blank'>http://lilybin.com</a>.");
```

Comment:
Can you please use gettext for this string as per:

_('string to be translated')

_('The Lilypond code is copied to clipboard. You can paste it here: ') + "<a href='http://lilybin.com' target='_blank'>http://lilybin.com</a>"

Reply:
Oops, forgot to do this!